src/frameworks/vite/index.ts

```diff
@@ -0,0 +1,59 @@
+import { execSync, spawn } from "child_process";
+import { existsSync } from "fs";
+import { copy } from "fs-extra";
+import { join } from "path";
+import { BuildResult, findDependency, FrameworkType, relativeRequire, SupportLevel } from "..";
+import { proxyRequestHandler } from "../../hosting/proxy";
+
+export const name = 'Vite';
+export const support = SupportLevel.Expirimental;
+export const type = FrameworkType.Toolchain;
```

Comment:
Now wondering about toolchain priority, should this come before the frameworks it can init?